builtin/providers/google/resource_compute_backend_service.go

```diff
@@ -0,0 +1,410 @@
+package google
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/hashcode"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
+)
+
+func resourceComputeBackendService() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeBackendServiceCreate,
+		Read:   resourceComputeBackendServiceRead,
+		Update: resourceComputeBackendServiceUpdate,
+		Delete: resourceComputeBackendServiceDelete,
+
+		Schema: map[string]*schema.Schema{
+			"backend": &schema.Schema{
+				Type: schema.TypeSet,
```

Comment:
Out of interest, is there any benefit in having it as a set instead of a list?  Does it help with computing diffs or does the backend not accept duplicates or something?

The cost is that you need that hash function :)